Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -558,19 +553,26 @@ private void UpdateGazeProvider()
             // another raycast if it's not populated
             if (gazeHitResult == null)
             {
-                // get 3d hit
-                hitResult3d.Clear();
-                var raycastProvider = CoreServices.InputSystem.RaycastProvider;
-                LayerMask[] prioritizedLayerMasks = (gazeProviderPointingData.Pointer.PrioritizedLayerMasksOverride ?? FocusLayerMasks);
-                QueryScene(gazeProviderPointingData.Pointer, raycastProvider, prioritizedLayerMasks,
-                    hitResult3d, maxQuerySceneResults, focusIndividualCompoundCollider);
+                if (gazeProviderPointingData != null)
```

Comment:
wouldn't we have to check for gazeProviderPointingData.Pointer != null as well?

Reply:
Ah good call!